src/shared/containers/search-box/SearchBox.js

```diff
@@ -30,6 +31,18 @@ class SearchBox extends Component {
 
     // ---------------------------------------------
 
+    _getRandomSuggestion() {
+        const suggestions = [
```

Comment:
Could we move this to the top to avoid creating the array whenever this function is called?